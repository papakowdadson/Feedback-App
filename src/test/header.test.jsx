import renderer from "react-test-renderer";
import Header from "../component/header";
import {screen} from '@testing-library/react'


describe('the nav bar ', () => { 
    it('testing header',()=>{
        const component=renderer.create(
        <Header  text='hi there' bgColor='blue' textColor='white'></Header>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    });
    
    it('testing words',()=>{
        renderer.create(
            <Header text='hi there'></Header>
        );
        const text =screen.findByText(/'hi there'/i);
        expect(text).toBeInTheDocument;
    });
 })
