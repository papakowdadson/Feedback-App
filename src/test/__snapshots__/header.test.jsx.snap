// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing header 1`] = `
<header
  style={
    Object {
      "backgroundColor": "blue",
      "color": "white",
    }
  }
>
  <div
    className="container"
  >
    <h2
      className="header-text"
    >
      hi there
    </h2>
  </div>
</header>
`;

exports[`the nav bar  testing header 1`] = `
<header
  style={
    Object {
      "backgroundColor": "blue",
      "color": "white",
    }
  }
>
  <div
    className="container"
  >
    <h2
      className="header-text"
    >
      hi there
    </h2>
  </div>
</header>
`;
